README.md

```diff
@@ -6,4 +6,55 @@ A bakery for use by Spinnaker to produce machine images.
 
 It presently supports producing Google Compute Engine images and AWS amis. It relies on packer and can be easily extended to support additional platforms.
 
-It exposes a REST api which can be experimented with via the Swagger UI: http://localhost:8087/swagger/index.html
+It exposes a REST api which can be experimented with via the Swagger UI: http://localhost:8087/swagger-ui.html
+
+# Developing rosco
+
+Need to run rosco locally for development? Here's what you need to setup and run:
+
+## Environment Setup
+```
+git clone git@github.com:spinnaker/rosco.git
+git clone https://github.com/spinnaker/spinnaker.git

```

Comment:
What I meant here was that they should both be either `git clone git@` or `git clone https://`, but not one of each.